docs/Drawer.md

```diff
@@ -78,34 +82,46 @@ import DrawerBody from '@pxblue/react-components/core/Drawer';
-
-| Attribute       | Description                             | Type               | Required | Default                      |
-|-----------------|-----------------------------------------|--------------------|----------|------------------------------|
-| active          | Is the item the current active item     | `boolean`          | no       | false                        |  
-| chevron         | Show chevron icon to the right          | `boolean`          | no       | false                        |  
-| divider         | Show a divider line below the item      | `boolean`          | no       | true                         |  
-| icon            | A component to render for the icon      | `React.Component`  | no       |                              |      
-| onClick         | A function to execute when clicked      | `function`         | no       |                              |    
-| statusColor     | Status stripe and icon color            | `string`           | no       |                              |    
-| subtitle        | The text to show on the second line     | `string`           | no       |                              |    
-| title           | The text to show on the first line      | `string`           | no       |                              |    
+| Prop Name             | Description                                                      | Type              | Required | Default |
+|-----------------------|------------------------------------------------------------------|-------------------|----------|---------|
+| activeBackgroundColor | Background color for the 'active' item                           | `string`          | no       |         |
+| activeFontColor       | Font color for the 'active' item                                 | `string`          | no       |         |
+| activeIconColor       | Icon color for the 'active' item                                 | `string`          | no       |         | 
+| activeItem            | itemID for the 'active' item                                     | `string`          | no       |         | 
+| backgroundColor       | The color used for the background                                | `string`          | no       |         |   
+| chevron               | Whether to have chevrons for all menu items                      | `boolean`         | no       |         |    
+| content               | Custom element, substitute for title                             | `React.Component` | no       |         |    
+| divider               | Whether to show a line between all items                         | `boolean`         | no       | true    |    
+| fontColor             | The color used for the text                                      | `string`          | no       |         |   
+| hidePadding           | Whether to hide the paddings reserved for menu item icons        | `boolean`         | no       |         | 
```

Comment:
does this need to be exposed?

Reply:
Resolve: These applies only to the top level items. When the users don't have any icons in their `items`, they may choose to hidePadding